python/xcfun.py

```diff
@@ -1,23 +1,45 @@
-#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+#
+# XCFun, an arbitrary order exchange-correlation library
+# Copyright (C) 2018 Ulf Ekström and contributors.
+#
+# This file is part of XCFun.
+#
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+#
+# For information on the complete list of contributors to the
+# XCFun library, see: <https://xcfun.readthedocs.io/>
+#
 
 import numpy
-from xcfun_swig import *
 
+from .xcfun_swig import *
 
-class XcFunException(Exception):
+
+class XCFunException(Exception):
     def __init__(self, msg, code=1):
         Exception.__init__(self, msg)
         self.error_code = code
 
 
+def xcfun_version():
```

Comment:
This wrapping can be removed if `from xcfun import *` is used in `__init__.py`.

Reply:
Fixed.